iXBRLViewerPlugin/viewer/src/js/ixbrlviewer.js

```diff
@@ -110,7 +110,12 @@ iXBRLViewer.prototype._reparentDocument = function () {
     /* Due to self-closing tags, our script tags may not be a direct child of
      * the body tag in an HTML DOM, so move them so that they are */
     $('body script').appendTo($('body'));
-    $('body').children().not("script").not('#ixv').not(iframeContainer).appendTo($(iframe).contents().find('body'));
+    const iframeBody = $(iframe).contents().find('body');
+    $('body').children().not("script").not('#ixv').not(iframeContainer).appendTo(iframeBody);
+
+    for (const bodyAttr of $('body').prop("attributes")) {
```

Comment:
Is it correct to copy all attributes, or should we only copy a subset (style, class)?

Should we "move" rather than "copy" the attributes (we only remove `style` below)?

Reply:
Both good questions :-)

The Python code doesn't add or remove any attributes to the `<body>` tag, so any attributes that are there will have come from the original iXBRL, so I think "move all" is probably what we want, as just about anything can potentially be used to influence styling. For example, an `id` attribute could be referenced in CSS.  Other attributes not currently permitted in iXBRL could also be used, such as `lang` and `data-` attributes.

Reply:
17fc967d55f4da5ab00c42d15eb969c427a1d35a implements moving all attributes.